pkg/operator/ceph/cluster/mon/mon.go

```diff
@@ -193,7 +194,7 @@ func (c *Cluster) Start(clusterInfo *cephclient.ClusterInfo, rookVersion string,
 	defer c.releaseOrchestrationLock()
 
 	clusterInfo.OwnerRef = c.ownerRef
-	c.ClusterInfo = clusterInfo
```

Comment:
It's still awkward that this `Start()` method takes a clusterInfo and assigns it to the `clusterInfo` field, even when the clusterInfo was already expected with `New()`.

Reply:
I agree, but I left that for future refactoring. I don't think this is necessary since all the references are pointers and should point to the same thing. But this makes it clear that the information is changing at least, versus it happening sort of magically.